src/components/general/paginator/Paginator.tsx

```diff
@@ -1,35 +1,30 @@
 /* Import Dependencies */
-import { useEffect, useState } from 'react';
-import { Pagination } from 'react-bootstrap';
+import { Pagination } from "react-bootstrap";
 
 
 /* Props Typing */
 interface Props {
     pageSize: number,
     pageNumber: number,
+    paginationRange: number[],
```

Comment:
Does this fix the issue with the pagination? I noticed that if you move to the end then it displays no more results. So you basically go beyond the items that are there. I also think we can show at least the next and previous page. Based on the `next` field in the json `links` you can see if there is a next page and display it.

Reply:
Yes, I modified the Paginator component so it will not longer continue going to pages without data.
Really? That is so much easier than the method I used in this commit!
Committed a new commit to fix this based upon the `links` data, omits the pagination range.